packages/editor-ui/src/components/TemplateList.vue

```diff
@@ -157,7 +157,11 @@ export default mixins(genericHelpers).extend({
 			inserted: (el, binding, vnode) => {
 				const f = () => {
 					if (vnode.context) {
-						if (vnode.context.$props.infinityScroll && vnode.context.$route.name === 'TemplatesView') {
+						if (
+							vnode.context.$props.infinityScroll &&
+							vnode.context.$route.name === 'TemplatesView' &&
```

Comment:
this should be a prop `infiniteScrollEnabled` instead based on the page we are on, instead of checking the route

Reply:
Updated as suggested and ```infiniteScrollEnabled``` prop refactored as suggested as well.